assets/templates/datasetLandingPage/static.tmpl

```diff
@@ -112,6 +112,16 @@
                     </section>
                 {{ end }}
                 <section>
+                    {{ if .DatasetLandingPage.Related.FilterableDatasets }}
+                    <div class="status status--blue margin-top--6">
+                        <p class="margin-top--0">Try the new way to filter and download this data:</p>
+                        <ul class="list--neutral margin-bottom--0">
+                            {{ range .DatasetLandingPage.Related.FilterableDatasets }}
+                            <li> <a href="{{ .URI }}">{{ .Title}}</a></li>
```

Comment:
Raising as we don't have an auto format correction tool for go .tmpl files at the moment. 
`{{ .Title}}` -> `{{ .Title }}` (space)
Tabbing everything between {{ if .DatasetLandingPage.Related.FilterableDatasets }} and {{ end }}